DependencyInjection/EkinoNewRelicExtension.php

```diff
@@ -108,5 +108,9 @@ public function load(array $configs, ContainerBuilder $container)
             ->replaceArgument(4, $transaction_naming_service)
             ->replaceArgument(5, $config['using_symfony_cache'])
         ;
+
+        if (!$config['twig_enabled']) {
```

Comment:
What's about checking if the class `Twig_Environment` exists?

Reply:
Agree, the flex way is to enable things when they are installed.

Reply:
Sure, will do